hcs-sxc-java-plugins/hcs-sxc-java-plugins-cryptography/pom.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0"?>
```

Comment:
Should rename plugin to indicate which type of encryption we use here to allow for different ones in the future

Reply:
the cryptography plugin contains both, encryption and keyrotation packages.  Best at this stage to rename at the package level.